File: Week02/Day2/swap_elements.py
# - Create a variable named `abc`
#   with the following content: `["first", "second", "third"]`
# - Swap the first and the third element of `abc`

abc = ["first", "second", "third"]

a, b = abc.index ("first"), abc.index("third")
abc[a], abc[b] = abc[b], abc[a]
print(abc)

File: Week07/Day3/swap_elements.js
'use strict';
// - Create a variable named `abc` with the following content: `["Arthur", "Boe", "Chloe"]`
// - Swap the first and the third element of `abc`

let abc = ["Arthur", "Boe", "Chloe"]
let tempAbc = abc
abc = [tempAbc[2], tempAbc[1], tempAbc[0]]


console.log(abc)